jsr352/core/pom.xml

```diff
@@ -80,8 +80,23 @@
         </testResources>
         <plugins>
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-jar-plugin</artifactId>
                 <executions>
+                    <execution>
+                        <id>default-jar</id>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>jar</goal>
+                        </goals>
+                        <configuration>
+                            <archive>
+                                <manifestEntries>
+                                    <Automatic-Module-Name>org.hibernate.search.jsr352</Automatic-Module-Name>
```

Comment:
Maybe rather `org.hibernate.search.jsr352.core`, since we need an additional module for JBeret.

By the way, maybe you should also add a module name to `hibernate-search-jsr352-jberet`? `org.hibernate.search.jsr352.jberet`, maybe?